dlrover/python/elastic_agent/torch/training.py

```diff
@@ -585,11 +584,13 @@ def run(self, role: str = DEFAULT_ROLE) -> bool:
             self._stop_workers(self._worker_group)
             if network_ready:
                 return True
-            elif i == 0 and self._worker_group.group_world_size <= 2:
-                logger.error(
-                    "Fail to check network when there are only 2 nodes."
-                )
-                raise RuntimeError("The node network is breakdown.")
+            else:
+                total_worker_num = len(self._client.get_running_nodes())
+                if total_worker_num <= 2:
```

Comment:
change this 2 to configurable?

Reply:
Not necessary and I have added annotation to explain the reason.